.github/workflows/main.yml

```diff
@@ -59,32 +72,78 @@ jobs:
         with:
           path: ${{github.workspace}}/vol-rest
 
-      - name: Autotools Configure + Build HDF5        
+      - name: CMake Configure + Build HDF5
+        if: matrix.build_system == 'cmake'
+        run: |
+          mkdir ${{github.workspace}}/hdf5/build
+          cd ./build
+          cmake \
+          -DHDF5_BUILD_HL_LIB=ON \
+          -DBUILD_SHARED_LIBS=ON -DHDF5_ENABLE_SZIP_SUPPORT=OFF \
+          -DHDF5_TEST_API=ON \
+          -DHDF5_ENABLE_Z_LIB_SUPPORT=OFF \
```

Comment:
Just a note, depending on which branch you're building this option could either be `Z_LIB` or `ZLIB` now. Though it shouldn't be a problem for develop as long as the default stays OFF